assembl/static2/js/app/main.jsx

```diff
@@ -27,13 +30,10 @@ class Main extends React.Component {
       const currentPhase = debateData.timeline.filter((phase) => {
         return phase.identifier === currentPhaseIdentifier;
       });
-      isRedirectionToV1 = currentPhase[0].interface_v1;
+      isRedirectionToV1 = currentPhase[0] && currentPhase[0].interface_v1;
     }
     this.props.addRedirectionToV1(isRedirectionToV1);
   }
-  componentDidMount() {
-    window.addEventListener('scroll', this.displayHeader);
```

Comment:
This isn't relevant any longer?

Reply:
this.displayHeader is undefined I can't see where it could be.

Reply:
there is no "displayHeader" string in the project code except here

Reply:
Okay, dead code. Good kill.